do_freebsd.sh

```diff
@@ -39,18 +31,24 @@ rm -rf build && ./do_cmake.sh "$*" \
 	-D WITH_SYSTEM_BOOST=ON \
 	-D WITH_LTTNG=OFF \
 	-D WITH_BLKID=OFF \
-	-D WITH_FUSE=OFF \
+	-D WITH_FUSE=ON \
 	-D WITH_KRBD=OFF \
 	-D WITH_XFS=OFF \
 	-D WITH_KVS=OFF \
-	-D WITH_MANPAGE=ON \
 	-D CEPH_MAN_DIR=man \
 	-D WITH_LIBCEPHFS=OFF \
 	-D WITH_CEPHFS=OFF \
 	-D WITH_EMBEDDED=OFF \
+	-D WITH_MGR=YES \
 	2>&1 | tee cmake.log
 
-cd build
-gmake -j$NPROC $BUILDOPTS
-gmake check 
+echo start building 
+date
+(cd build; gmake -j$NPROC $BUILDOPTS )
+(cd build; gmake -j$NPROC $BUILDOPTS ceph-disk)
+(cd build; gmake -j$NPROC $BUILDOPTS ceph-detect-init)
+
+echo start testing 
+date
+(cd build; ctest -j$NPROC || ctest --rerun-failed --output-on-failure)
```

Comment:
@wjwithagen "make tests" does not **run** tests. it just builds the necessary bits for running tests. anyway, it's just my 2 cents.